src/merge_request.rs

```diff
@@ -182,14 +182,15 @@ pub async fn cleanup_merge_request(
 		MergeRequestCleanupReason::Error
 		| MergeRequestCleanupReason::Cancelled => {
 			for dependent in related_dependents.values() {
-				let _ = cleanup_merge_request(
+				let result = cleanup_merge_request(
 					state,
 					&dependent.sha,
 					&dependent.owner,
 					&dependent.repo,
 					dependent.number,
 					reason,
 				);
+				drop(result)
```

Comment:
We do actually want the cleanups to be performed, so this should be let _ = cleanup_merge_request(..).await

Reply:
I thought that the cleanups are still performed, and the `drop` just makes dropping the result of it explicit.

Reply:
No, rust futures don't actually do anything until they're polled.